### PR TITLE
Prepare 6.0.2

### PR DESCRIPTION
**Pull request by radrow**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 ### Removed
 
+## [6.0.2] 2021-07-05
+### Changed
+- `List.from_to_step` now forbids non-positive step
+- Fixed leaking state between contracts
```
Comment (hanssv) [inline CHANGELOG.md:15]: That sounds a bit worrying, but hopefully those reading it realize this is the compiler and this bug doesn't affect on-chain contracts 😅

**Review by hanssv — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (radrow) [reply CHANGELOG.md:16]: aeh, mentioned

Comment (hanssv) [reply CHANGELOG.md:16]: Was more thinking about "leaking state between contracts" but fair enough 😅

**Review by hanssv — APPROVED** [review]

**Comment by radrow** [comment]
Nevermind, let's leave it as it is.

**Review by UlfNorell — APPROVED** [review]

**Pull request merged**